packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -49,12 +49,50 @@ export type PODPCDRecordArg = RecordContainerArgument<
   PODPCDArgValidatorParams
 >;
 
+/**
+ * Type encapsulating prescribed values for PODs to be fed into a GPCPCD,
```

Comment:
This description doesn't really specify what these values are for.  Maybe something like this?

"User inputs to a proof will be filtered to only those PODs which contain matching values."